api/apps/api/test/geo-features/geo-feature-tags.e2e-spec.ts

```diff
@@ -0,0 +1,354 @@
+import { HttpStatus } from '@nestjs/common';
+import * as request from 'supertest';
+import { GeoFeature } from '@marxan-api/modules/geo-features/geo-feature.api.entity';
+import { bootstrapApplication } from '../utils/api-application';
+import { GivenUserIsLoggedIn } from '../steps/given-user-is-logged-in';
+import { GeoFeatureTag } from '@marxan-api/modules/geo-feature-tags/geo-feature-tag.api.entity';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { OrganizationsTestUtils } from '../utils/organizations.test.utils';
+import { E2E_CONFIG } from '../e2e.config';
+import { ProjectsTestUtils } from '../utils/projects.test.utils';
+import { Project } from '@marxan-api/modules/projects/project.api.entity';
+import { Organization } from '@marxan-api/modules/organizations/organization.api.entity';
+import { FixtureType } from '@marxan/utils/tests/fixture-type';
+import { AppConfig } from '@marxan-api/utils/config.utils';
+import { v4 } from 'uuid';
+
+let fixtures: FixtureType<typeof getFixtures>;
+
+const tagMaxlength = AppConfig.get<number>('marxan.maxTagLength');
+
+describe('GeoFeatureTag PATCH (e2e)', () => {
```

Comment:
I forgot - as this is a new test folder within `/api/apps/api/test`, we also need to add this to the job matrix in `.github/workflows/tests-api-e2e.yml`, otherwise these tests will never be executed.

I think I should figure out a pre-push hook to check that we don't forget this when creating new folders within the root test folder for either api or geoprocessing.

what I mean, in practice is something like this:

```
diff --git a/.github/workflows/tests-api-e2e.yml b/.github/workflows/tests-api-e2e.yml
index 285aa8a51..fb9451765 100644
--- a/.github/workflows/tests-api-e2e.yml
+++ b/.github/workflows/tests-api-e2e.yml
@@ -25,6 +25,7 @@ jobs:
           - 'calculate-pu-protection-level'
           - 'cloning'
           - 'files-repository'
+          - 'geo-features'
           - 'implicit-permissions'
           - 'integration'
           - 'legacy-project-import'
```